go.mod

```diff
@@ -3,23 +3,17 @@ module github.com/vmware-tanzu/tanzu-framework
 go 1.16
 
 replace (
-	github.com/go-logr/logr => github.com/go-logr/logr v0.1.0
-	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.3.1 // indirect
-	k8s.io/api => k8s.io/api v0.17.11
-	k8s.io/apimachinery => k8s.io/apimachinery v0.17.11
-	k8s.io/client-go => k8s.io/client-go v0.17.11
-	k8s.io/klog/v2 => k8s.io/klog/v2 v2.0.0
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.5.14
+	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5 // indirect
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.0
```

Comment:
same comment as earlier

Reply:
I think this is required. Please refer to https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2467